app/components/RelatedRecordTableRow.tsx

```diff
@@ -138,6 +161,15 @@ export default function RelatedRecordTableRow<
             </td>
           );
         })}
+        {RowActionButton && (
+          <td>
```

Comment:
could you add a `text-align: right` here? so we can have this aligned to the top button 

<img width="691" alt="Screenshot 2025-02-04 at 9 41 02 AM" src="https://github.com/user-attachments/assets/52741c44-40cc-4b41-8606-f8ac01306f2b" />

Reply:
`text-end` did the trick 🙏